### DISCUSSION TITLE
[Crowdin] Add Slovak (Slovakia) language

### DISCUSSION THREAD

**Discussion by bachelor-emgi** [RaspAP/raspap-webgui#1369] [Ideas]
Hey I am kinda bored till there is a new build that supports Ubuntu 23.04. So I have an idea that would at least fill my bored days hours before the new version releases.

**Comment by billz** [comment]
Thanks for offering to translate Slovak! I've added the language to our [localization project](https://crowdin.com/translate/raspap/10/en-sk?filter=basic&value=3). 
Any questions, feel free to reply here or message me privately on Crowdin.

**Comment by billz** [comment]
Re: Ubuntu 23.04, you're welcome to test the PR 😉
just be aware that this distro requires some pre-install steps https://github.com/RaspAP/raspap-webgui/pull/1354#issuecomment-1575455634

**Reply by bachelor-emgi** [reply]
Thanks, will definitely try it. Does also insiders have it merged? Can't see any dedicated branch for it.

EDIT: Tried it and works perfectly, just cant make work bridged AP mode since it is grayed out but at least works

**Reply by billz** [reply]
Awesome, thanks for testing. Re: insiders, the branch will be merged here first then applied to that repo - just need to update our docs accordingly. New releases should be available this coming week.
